### PR TITLE
context for notifications

### PR DESCRIPTION
**Pull request by oliwia1003**
Toast state added to context layer.

### CODE REVIEW

File: src/providers/LayerProvider/LayerProvider.tsx [hunk 1]
```diff
@@ -98,24 +114,28 @@ export const LayerProvider = ({
     usePlaidSandbox,
     onboardingStep: undefined,
     environment,
+    toasts: [],
   })
 
+  // const [toasts, setToasts] = useState<(ToastProps & { isExiting: boolean })[]>([]);
```
Comment (tomaszantas) [inline src/providers/LayerProvider/LayerProvider.tsx:120]: ✂️

File: src/types/layer_context.ts [hunk 2]
```diff
@@ -1,107 +1,125 @@
-import { Business, Category } from '../types'
-import { ExpiringOAuthResponse } from './authentication'
+import { ToastProps } from "../components/Toast/Toast";
```
Comment (tomaszantas) [inline src/types/layer_context.ts:1]: Possibly some eslint/prettier issue here... we don't use semicolons `;`

**Review by tomaszantas — COMMENTED** [review]
Looks good in overall, but there is some prettier / eslint issue.
Take a look pls also on #156 <- I've added a way to dismiss toast on click. If all there looks good, feel free to merge (it's targeted to your branch, so after merging PR on Github, you may need to pull changes locally before making any other adjustments)

**Review by ArkuVonSymfon — COMMENTED** [review]
Nothing more than prettier

**Pull request merged**